### DISCUSSION TITLE
New Shredos.x86_64 to be released in next 24hrs

### DISCUSSION THREAD

**Discussion by PartialVolume** [PartialVolume/shredos.x86_64#62] [Announcements]
A new version of shredos is being released in the next 24 hrs. Here's the draft release notes. Please note that the current shredos based on buildroot 2020.05 will be renamed as [https://github.com/PartialVolume/shredos.x86_64_2020.05](https://github.com/PartialVolume/shredos.x86_64_2020.05). The latest version of shredos will always be located at [https://github.com/PartialVolume/shredos.x86_64](https://github.com/PartialVolume/shredos.x86_64)

------

v2021.08.2_20_x86-64_0.32.014

| Buildroot Version | Kernel Version | Shredos Minor version | Architecture | Nwipe Version |
| ----------------------- | ------------------- | ------------------------------- | ---------------- | ------------------- |
| 2021.08.2             | 5.13.19           |  020                               | x86-64          | 0.32.014        |

Other software versions used within ShredOS for disc related operations
| hexedit | hdparm | Smartctl |
| ---------- | ---------- | -------------------- |
|    1.5     |   9.62    |           7.2          |

## Notable changes in this release

- New buildroot release v2021.08.2, based on the 5.13.19 kernel.
- Upgraded nwipe from 0.32.003 to 0.32.014 master.
- Better UEFI support.
  - bootx64.efi has been rebuild with the extra grub modules efi_gop, efi_uga and progress.
  - During UEFI boot, the `Recalbox` message has been replaced with the message `booting shredos` along with a progress display as shredos.img is read from the USB device.
  - Fixes issues on UEFI systems that booted shredos but there was no display.
- Upgraded hdparm from v9.58 to v9.62
- Upgraded smartctl (smartmontools) from v7.1 to v7.2
- Tidied up the github commit history. As we are not interested in the buildroot commit history only shredos commits you will find the github commit history starts with a initial commit on this new buildroot version.
- Past repositories of shredos based on older kernels will now be renamed as shredos.x86_64_[build root version], i.e [https://github.com/PartialVolume/shredos.x86_64_2020.05](https://github.com/PartialVolume/shredos.x86_64_2020.05)
- The latest shredos URL will always be [https://github.com/PartialVolume/shredos.x86_64](https://github.com/PartialVolume/shredos.x86_64) for 64 bit systems and [https://github.com/PartialVolume/shredos.i686](https://github.com/PartialVolume/shredos.i686) for 32 bit systems.

### Nwipe changes from 0.32 to 0.32.014
- [Fix slight screen corruption on 80 column display #348](https://github.com/martijnvanbrummelen/nwipe/commit/0ee4099c641548b6a6120b210c1af2469e132cc7)
- [Add SAS to GUI. #350](https://github.com/martijnvanbrummelen/nwipe/pull/350)
- Corrections to typos using codespell #351-353 [Typos found by codespell #351](https://github.com/martijnvanbrummelen/nwipe/pull/351)
- [Check smartctl for unresolved bus types SATA #358](https://github.com/martijnvanbrummelen/nwipe/pull/358)
- More typo corrections [Update device.c #356](https://github.com/martijnvanbrummelen/nwipe/pull/356)
- [Add drive temperature monitoring #360](https://github.com/martijnvanbrummelen/nwipe/pull/360)
- [Added temperature to drive selection window #361](https://github.com/martijnvanbrummelen/nwipe/pull/361)
- [Update README.md re. netboot.xyz #363](https://github.com/martijnvanbrummelen/nwipe/pull/363/commits)
- [Moved the temperature readout #364](https://github.com/martijnvanbrummelen/nwipe/pull/364)
- [Remove /dev/ from gui for long filenames like nvme0n1 to maintain column alignment #365](https://github.com/martijnvanbrummelen/nwipe/pull/365)
- [Add -q --quiet option - anonymize serial numbers and other identifiable data #366](https://github.com/martijnvanbrummelen/nwipe/pull/366)
- [Anonymize SMBIOS/DMI data if -q --quiet specified #367](https://github.com/martijnvanbrummelen/nwipe/pull/367)
- [Anonymize 'LU WWN ID' when -q specified. #371](https://github.com/martijnvanbrummelen/nwipe/pull/371)
- [Fix summary table - Control-C #373](https://github.com/martijnvanbrummelen/nwipe/pull/373)
- [Make verify & pass error succinct #374](https://github.com/martijnvanbrummelen/nwipe/pull/374)
- [Fixes ETA under fault condition #375](https://github.com/martijnvanbrummelen/nwipe/pull/375)
- [Add temperature support to NVME drives #377](https://github.com/martijnvanbrummelen/nwipe/pull/377)
- [Distinguish between a wipe & verify #378](https://github.com/martijnvanbrummelen/nwipe/pull/378)
- [Make it clear we are in anonymized mode #379](https://github.com/martijnvanbrummelen/nwipe/pull/379)
- [Update nvme temperatures. #380](https://github.com/martijnvanbrummelen/nwipe/pull/380)
- [Update temperature for some nvme devices #381](https://github.com/martijnvanbrummelen/nwipe/pull/381)
- [Update options.c with additional infor for -q #383](https://github.com/martijnvanbrummelen/nwipe/pull/383)

## Known issues
- hexedit help screen doesn't work. This is due to groff (nroff) missing and not being available in either buildroot or busybox. I'm hoping to eventually pull this package in from github. If anybody would like to submit a PR for this, please feel free.
- In legacy mode shredos's default terminal resolution is 640x480, occasionally you may come across a monitor that will not display this resolution, shredos has booted but you can't see it. This can be corrected by changing the default shredos resolution in legacy mode to 1024x768, most monitors appear to accept that resolution. This is accomplished by editing the grub.cfg file. See [how-to-make-a-persistent-change-to-the-terminal-resolution](https://github.com/PartialVolume/shredos.x86_64#how-to-make-a-persistent-change-to-the-terminal-resolution). The default 640x480 legacy resolution may be changed to 1024x768 in a future release if requests are made for this change. UEFI resolution tends to be the native resolution of the monitor or no lower than 1024x768.

**Comment by Firminator** [comment]
Lots of work went into this release. Thanks for putting this documentation together.

**Comment by PartialVolume** [comment]
@Firminator Thanks for the help testing. It will be released with both .iso and .img formats in 64bit. 32bit to follow.
